homeassistant/components/google/__init__.py

```diff
@@ -93,7 +130,7 @@ def do_authentication(hass, hass_config, config):
     oauth = OAuth2WebServerFlow(
         client_id=config[CONF_CLIENT_ID],
         client_secret=config[CONF_CLIENT_SECRET],
-        scope='https://www.googleapis.com/auth/calendar.readonly',
+        scope='https://www.googleapis.com/auth/calendar',
```

Comment:
Why don't we reuse `SCOPES` ?

Reply:
Hm, I didn't think about that, just changed both. Change this to be `scope=SCOPES`?